src/editor/objects/Image.js

```diff
@@ -16,15 +15,25 @@ export const ImageProjection = {
   Equirectangular360: "360-equirectangular"
 };
 
+export const ImageTransparencyMode = {
+  None: "none",
+  Alpha: "alpha",
+  Cutout: "cutout"
```

Comment:
I'd rather we use the term alpha "mask" and "blend" as they correspond to the terminology already used in glTF: https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#alpha-coverage

Reply:
Makes sense, I changed it to `alphaMode` and changed the possible values to `opaque`, `blend`, and `mask` to align more with the gltf spec